reversers/database-synchroniser/plugins/org.obeonetwork.dsl.database.gen.common/src/org/obeonetwork/dsl/database/gen/common/services/StatusUtils.java

```diff
@@ -0,0 +1,77 @@
+/*******************************************************************************
```

Comment:
What do you think putting this class in here ? org.obeonetwork.dsl.is

Reply:
I will do it in a subsequent commit